src/main/java/mod/gate/mixin/InGameHudMixin.java

```diff
@@ -0,0 +1,35 @@
+package mod.gate.mixin;
+
+
+import mod.gate.events.ActionbarReceiveEvent;
+import mod.gate.events.EventHandler;
+import net.minecraft.client.gui.hud.InGameHud;
+import net.minecraft.text.Text;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(InGameHud.class)
+public class InGameHudMixin {
```

Comment:
DOn't call it `InGameHudMixin` but rather `ActionBarMixin` since it doesn't handle the entire class and the second is more descriptive